rotkehlchen/tests/unit/decoders/test_aave_v3.py

```diff
@@ -1943,3 +1944,88 @@ def test_gnosis_xdai_deposit(gnosis_inquirer, gnosis_accounts) -> None:
         notes=f'Receive {interest_amount} aGnoWXDAI as interest earned from AAVE v3',
         counterparty=CPT_AAVE_V3,
     )]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('base_accounts', [['0xE37b28362F65060C18c16398cFD23275D8CaE750']])
+def test_aave_v3_collateral_swap(base_inquirer, base_accounts) -> None:
+    tx_hash = deserialize_evm_tx_hash('0xdc1a92c8cbda2fe7917e633efd889d17fc62e88e0f584af65f577b5d2a8bcb3c')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=base_inquirer, tx_hash=tx_hash)
+    expected_events = [
+        EvmEvent(
```

Comment:
```suggestion
    expected_events = [EvmEvent(
```
can save a couple lines and de-indent the list of events. Also could just do the assertion right here instead of at the end as well.